### DISCUSSION TITLE
Linting

### DISCUSSION THREAD

**Discussion by MattIPv4** [alveusgg/extension#35] [Ideas]
👋 What're folks thoughts on linting?

There seems to be a lot of inconsistency in the code style throughout the extension, even with basic things like whether to have a space between the start of a comment and the text in the comment, or space between a selector and its brackets.

I've generally liked to lean on eslint + stylelint, with airbnb as my default config for eslint and just relying on stylelint's standard config for SCSS.

I'd also like to add an editorconfig file (no linting), just to encourage folks' IDEs to be consistent about having newlines at the end of files, and not leaving trailing whitespace on lines.

**Comment by abdullahmorrison** [comment]
Been procrastinating adding this. It should be done. #36
